lib/composables/Lang.ts

```diff
@@ -0,0 +1,32 @@
+import { inject, provide } from 'vue'
+
+export type Lang = 'en' | 'ja'
+
+export interface Trans<T> {
+  t: T
+}
+
+export interface TransMessages<T> {
+  en: T
+  ja: T
+}
+
+export const SefirotLangKey = 'sefirot-lang-key'
+
+export function provideLang(lang: Lang) {
+  provide(SefirotLangKey, lang)
+}
+
+export function useLang(): Lang {
+  return inject(SefirotLangKey) ?? 'en'
```

Comment:
```suggestion
  return inject(SefirotLangKey, 'en')
```

it'll throw otherwise